_data/navbars/maintenance.yml

```diff
@@ -29,7 +29,7 @@ section:
     path: /maintenance/ebpf/enabling-bpf
   - title: eBPF on EKS
     path: /maintenance/ebpf/ebpf-and-eks
-  - title: Troubleshooting
+  - title: troubleshooting
```

Comment:
is this right? The rest of our doc titles are all capitalized I think

Reply:
That is a mistake. Fixed.

Reply:
Fixed.